CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+Version 2.0
+
+# Bundler-inject Changelog
+
+Doing our best at supporting [SemVer](http://semver.org/) with
+a nice looking [Changelog](http://keepachangelog.com).
+
+## Version [HEAD] <sub><sup>now</sub></sup>
+
+## Version [2.0.0] <sub><sup>2021-05-01</sub></sup>
+
+* Drops support for bundler below 2.0
+* Drops support for ruby below 2.6
```

Comment:
Let's mark these as breaking changes to be clear why a 2.0 was cut

```suggestion
* **BREAKING**: Drops support for bundler below 2.0
* **BREAKING**: Drops support for Ruby below 2.6
```

While not in keep-a-changelog directly, this is general consensus for how to do it in one of the issues in their issue tracker.  Additionally, this is how we do it in other gems like more_core_extensions.